src/ProjectService.Models.Dto/Requests/CreateProjectRequest.cs

```diff
@@ -14,5 +14,6 @@ public class ProjectRequest
         public Guid? DepartmentId { get; set; }
         public ProjectStatusType Status { get; set; }
         public IEnumerable<ProjectUserRequest> Users { get; set; }
+        public IEnumerable<ProjectImage> ProjectsImages { get; set; }
```

Comment:
you create one project with many images => ProjectImages

Reply:
done